x-pack/plugins/beats_management/public/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -0,0 +1,184 @@
+/*
```

Comment:
This file was moved, but I did delete significant amount of code here around mounting and interfacing with Angular